modules/document_repository/jsx/docIndex.js

```diff
@@ -126,7 +126,7 @@ class DocIndex extends React.Component {
     let result = <td>{cell}</td>;
     switch (column) {
       case 'File Name':
-        let downloadURL = loris.BaseURL + '/document_repository/Files/' + encodeURIComponent(row['File Name']);
+        let downloadURL = loris.BaseURL + '/document_repository/Files/'+ row['Uploaded By'] +'/' + encodeURIComponent(row['File Name']);
```

Comment:
I don't understand this.. why is uploaded by part of the URL?

Reply:
I want to pass a username as a parameter by the front-end clicking.
If I get the username , I could locate the file which upload by users.

Reply:
I think a better way to do this would be to send the ID from the front-end and use it to query the database for the `Data_dir` value which contains the file location.

Reply:
api /Files/ID has been used to get all the upload info of this file.

Reply:
api /Files/filename will download the file. I will remove the Uploaded By from url.